certlogic/certlogic-dart/lib/src/evaluate.dart

```diff
@@ -4,8 +4,10 @@ import 'package:certlogic_dart/src/typings.dart';
 /// The library, you can use the static functions, so you don't need to instantiate this class
 class CertLogic {
   static dynamic _accumulator;
+  static dynamic _current;
+  static dynamic _data;
 
-  /// Evaluate a variable
+  /// _evaluate a variable
```

Comment:
I suspect this is a textual-find-replace-thingy ;)

Reply:
Yes, good find!